hw/ip/otbn/data/otbn.hjson

```diff
@@ -134,17 +134,34 @@
   registers: [
     { name: "CMD"
       desc: "command register",
-      swaccess: "r0w1c",
+      swaccess: "wo",
       hwaccess: "hro",
       hwext: "true",
       hwqe:  "true",
       fields: [
-        { bits: "0"
-          name: "start"
+        { bits: "1:0"
+          name: "cmd"
+          resval: 0,
           desc: '''
-            Start the operation
+            Initiates an OTBN operation.
+
+            1 (EXECUTE): Start the execution of the program stored in the
+               instruction memory, starting at address !!START_ADDR.
+
+            2 (SEC_WIPE_DMEM): Securely remove all contents from the data
+              memory.
 
-            The completion is signalled by the done interrupt.
+            3 (SEC_WIPE_IMEM): Securely remove all contents from the instruction
+              memory.
+
+            During the operation the command is reflected in the !!STATUS
+            register, indicating that OTBN is busy performing the requested
+            operation.
+
+            All commands signal their completion by raising the done interrupt;
+            alternatively, software may poll the !!STATUS register.
+
+            Unrecognized commands are ignored.
```

Comment:
Can we make it more obvious the numbers given are what you write to cmd to perform the operation?

Reply:
Yeah, I need to work on the formatting and check what other IP blocks are doing.

Reply:
I reworked this list into a table.